mage_ai/data_cleaner/shared/utils.py

```diff
@@ -19,21 +20,22 @@ def clean_series(series, column_type, dropna=True):
         return series_cleaned
     
     first_item = series_cleaned.dropna().iloc[0]
-    if column_type == NUMBER:
+    if column_type == NUMBER or column_type == NUMBER_WITH_DECIMALS:
+        is_percent = False
         if type(first_item) is str:
             series_cleaned = series_cleaned.str.replace(',', '')
-            if series_cleaned.str.contains(CURRENCY_SYMBOLS).sum() != 0:
-                return # handle this case with reformat transform action
+            if series_cleaned.str.count(CURRENCY_SYMBOLS).sum() != 0:
```

Comment:
Does the reformat_values cleaning rule still work?

Reply:
Yes. that make sense. To make the reformat_values rule work, we'll need to pass the uncleaned dataframe to the cleaning rule as well.

Reply:
I'll start working on rewriting some of the cleaning rule code so both the cleaned and uncleaned dataframes are given as inputs. In this case then, we will only be showing the uncleaned dataframe to the user, correct?